galaxy.yml

```diff
@@ -54,6 +54,9 @@ tags:
 # range specifiers can be set and are separated by ','
 dependencies:
   ansible.posix: "*"
+  community.crypto: "*"
+  community.general: "*"
+  community.postgres: "*"
```

Comment:
adding dependencies here will make `ansible-galaxy` install the dependencies, but some users don't use `pulp_database` role, therefore they would be installing `community.postgres` for nothing.
Not sure what would be the best way to deal with these "conditional dependencies"

Reply:
@fao89 Good point. I'm very curious if there is a way.

We do declare dependencies in the roles, maybe that can help.

Reply:
I'm wondering: should we vendor these "conditional dependencies"?